command/test-fixtures/hcl2_upgrade_basic/input.json

```diff
@@ -5,7 +5,9 @@
         "aws_region": null,
         "aws_secondary_region": "{{ env `AWS_DEFAULT_REGION` }}",
         "aws_secret_key": "",
-        "aws_access_key": ""
+        "aws_access_key": "",
+        "password": "{{ aws_secretsmanager `sample/app/password` }}",
+        "password_key": "{{ aws_secretsmanager `sample/app/passwords` `api_key` }}"
```

Comment:
What happens in the case I have :

        "password": "{{ aws_secretsmanager `sample/app/password_part_1` }}{{ aws_secretsmanager `sample/app/password_part_2` }}",
        "password_key": "SURPRISE!{{ aws_secretsmanager `sample/app/passwords` `api_key` }}"

Reply:
first: I think the second value will override the first 🤔 
second: `SURPRISE!` will get lost 

Ok need to think about these two situations.

Reply:
Gotcha, I think that it would be okay to go simple and just generate a datasource name everytime we see a different aws_secretsmanager.

Another (maybe a bit complex?) thing we could do is keep the variable name and make a local out of these.

Reply:
I actually think that the second option is the only one.
Imagine this :
``` 
variables: {
"password": "{{ aws_secretsmanager `sample/app/password_part_1` }}{{ aws_secretsmanager `sample/app/password_part_2` }}"
} 
``` 
will be updated to a variable like:
```
 password = "${data.aws_secretsmanager.autogenerated_0.value}${data.aws_secretsmanager.autogenerated_1.value}",
 ```
which won't work because data sources don't work on variables. So I need to make it a local and keep this variable-to-local change information somewhere, to transform the variable `password` usage to the local one.

Reply:
Got it ! 👍🏼  Good luck with that one 🙂